articles/relnotes/index.md

```diff
@@ -17,19 +17,33 @@ For installation instructions, please refer to the [install guide](xref:microsof
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
 
+## Version 0.xx.xxxx.xxxx
+
+*Release date: February 27th, 2020*
+
+This release contains the following:
+
+- New Quantum Machine Learning library, for more information go to our [QML docs page](https://docs.microsoft.com/quantum/libraries/machine-learning/?view=qsharp-preview)
```

Comment:
The 10×–20× speedup is indeed an IQ# issue, and affects loading of packages; that is, it's not a general speedup. Since the katas have not been updated to the newest version of IQ#, there may not be a change in performance there for this release.

Reply:
Define "for this release" :-) It will be once we update the katas, for which the release needs to be out.